docs/get-started/README.md

```diff
@@ -1,73 +1,85 @@
+`ReactMapGL` requires `node >= v4` and `react >= 15.4`.
 
+```sh
+npm install --save react-map-gl
+```
+or
+```sh
+yarn add react-map-gl
+```
 
-# Installation
+## Example
+
+```js
+import {Component} from 'react';
+import ReactMapGL from 'react-map-gl';
+
+class Map extends Component {
+  render() {
+    return (
+      <ReactMapGL
+        width={400}
+        height={400}
+        latitude={37.7577}
+        longitude={-122.4376}
+        zoom={8}
+        onViewportChange={(viewport) => {
+          const {latitude, longitude, zoom} = viewport;
+          // Optionally call `setState` and use the state to update the map.
+        }}
+      />
+    );
+  }
+}
+```
 
-react-map-gl needs to be installed in a product that also requires react 0.14.x and ImmutableJS 3.x. (These are referred to as its [peerDependencies](https://nodejs.org/en/blog/npm/peer-dependencies/)) so if you haven't already, you'll need to run the following in your project:
+## Server Side Rendering
 
-    npm install react
-    npm install immutable
-    npm install r-dom
+`ReactMapGL` depends on `gl`, which may cause issues when running in a server
+environment without `gl` installed. You can either make sure that your system
+has `gl` installed or use the following work-around to ensure you only require
+`ReactMapGL` on the client.
 
-And then install react-map-gl with:
+```js
+// is-browser is an npm package, but you can use any other solution to make sure
+// that you are in a browser environment.
+import isBrowser from 'is-browser';
 
-    npm install react-map-gl
+const ReactMapGL = 'div';
+if (isBrowser) {
+  ReactMapGL = require('react-map-gl').default;
+}
+```
 
-Here's a preview of the API and how you could create a simple non-interactive map.
+## Using with Browserify, Webpack, and other environments
 
-```ks
-var MapGL = require(\'react-map-gl\');
-  // later in your app...
-  <MapGL
-    width this.state.map.width + '}
-    height this.state.map.height + '}
-    latitude d3.round(this.state.map.latitude, 3) + '}
-    longitude d3.round(this.state.map.longitude, 3) + '}
-    zoom d3.round(this.state.map.zoom, 3) + '}
-    mapStyle={mapStyle
-   />
-```
+* `browserify` - `ReactMapGL` is extensively tested with `browserify` and works
+without issue.
 
-TODO - live example
-```
-      r(MapGL,
-        assign({onChangeViewport: this._onChangeViewport}, this.state.map),
-        r(Attribute, this.state.map)
-      ),
-```
+* `webpack 1` - look at the [deck.gl exhibits](https://github.com/uber/deck.gl/tree/master/exhibits)
+folder, demonstrating a working demo using `webpack`.
 
-# mapStyle prop
+* `webpack 2` - `ReactMapGL` is built using `webpack 2` and works without issue.
```

Comment:
This is not completely true, the app needs to at least configure the alias to the mapbox dist, which also for instance breaks `create-react-app`